### DISCUSSION TITLE
Should the dashboard be showing JPY as USD?

### DISCUSSION THREAD

**Discussion by z0rg0n** [firefly-iii/firefly-iii#9235] [General]
I brought it up [here](https://github.com/orgs/firefly-iii/discussions/5291) but I guess that question has been answered so I want to ask it more broadly.

Should other currencies be converted somehow in the dashboard? On my dashboard JPY is showing as USD which throws off my chart.

**Comment by JC5** [comment]
Hey, great question!

This is on my list of things to improve.

**Comment by franco-ruggeri** [comment]
Hi all, I started using Firefly III recently, and I believe this feature is key. I can't really analyze my financial situation without currency conversion in the dashboard/reports. Currencies should be converted to the default currency using exchange rates.
